### ISSUE TITLE
Unrecognized options in ~/.fleek.yml cause fleek to report "configuration files not found"

### ISSUE THREAD

**Issue by tml** [ublue-os/fleek#212]
**Describe the bug**
In attempting to configure home-manager's ssh module in my `~/.fleek.yml`, I found that inserting configuration commands fleek didn't recognize would cause it to report "configuration files not found"

**To Reproduce**
In `~/.fleek.yml` add the following to your `programs` key:

```programs:
    - ssh:
      controlMaster: yes
```

**Expected behavior**
Nothing at worst; configuration of the `ssh` program in home-manager at best. Though I now know that `user.nix` is the correct way to configure this, it is confusing what items I should be doing in `~/.fleek.yml` and which I should be doing in `user.nix`. But in neither case am I expecting `configuration files not found`.

**Environment**
 - OS: MacOS Venture
 - Shell: bash
 - fleek version: 0.9.5

**Additional context**
```FLEEK_DEBUG=1 fleek apply
 [d]  debug enabled
 [d]  go  go1.19.6
      path  github.com/ublue-os/fleek/cmd/fleek
      mod github.com/ublue-os/fleek (devel)
      dep atomicgo.dev/cursor v0.1.1
      dep atomicgo.dev/keyboard v0.2.9
      dep atomicgo.dev/schedule v0.0.2
      dep github.com/ProtonMail/go-crypto v0.0.0-20230518184743-7afd39499903
      dep github.com/atotto/clipboard v0.1.4
      dep github.com/aymanbagabas/go-osc52/v2 v2.0.1
      dep github.com/charmbracelet/bubbles  v0.14.0
      dep github.com/charmbracelet/bubbletea  v0.23.1
      dep github.com/charmbracelet/lipgloss v0.7.1
      dep github.com/cloudflare/circl v1.3.3
      dep github.com/containerd/console v1.0.3
      dep github.com/emirpasic/gods v1.18.1
      dep github.com/erikgeiser/promptkit v0.8.0
      dep github.com/fatih/color  v1.15.0
      dep github.com/fitv/go-i18n v1.0.4
      dep github.com/go-git/gcfg  v1.5.1-0.20230307220236-3a3c6141e376
      dep github.com/go-git/go-billy/v5 v5.4.1
      dep github.com/go-git/go-git/v5 v5.7.0
      dep github.com/golang/groupcache  v0.0.0-20210331224755-41bb18bfe9da
      dep github.com/google/uuid  v1.3.0
      dep github.com/gookit/color v1.5.3
      dep github.com/hashicorp/go-version v1.6.0
      dep github.com/imdario/mergo  v0.3.15
      dep github.com/jbenet/go-context  v0.0.0-20150711004518-d14ea06fba99
      dep github.com/kevinburke/ssh_config  v1.2.0
      dep github.com/lithammer/fuzzysearch  v1.1.8
      dep github.com/lucasb-eyer/go-colorful  v1.2.0
      dep github.com/mattn/go-colorable v0.1.13
      dep github.com/mattn/go-isatty  v0.0.17
      dep github.com/mattn/go-localereader  v0.0.1
      dep github.com/mattn/go-runewidth v0.0.14
      dep github.com/muesli/ansi  v0.0.0-20221106050444-61f0cd9a192a
      dep github.com/muesli/cancelreader  v0.2.2
      dep github.com/muesli/mango v0.1.0
      dep github.com/muesli/mango-cobra v1.2.0
      dep github.com/muesli/mango-pflag v0.1.0
      dep github.com/muesli/reflow  v0.3.0
      dep github.com/muesli/roff  v0.1.0
      dep github.com/muesli/termenv v0.15.1
      dep github.com/otiai10/copy v1.11.0
      dep github.com/pjbgf/sha1cd v0.3.0
      dep github.com/pkg/errors v0.9.1
      dep github.com/pterm/pterm  v0.12.62
      dep github.com/rivo/uniseg  v0.4.4
      dep github.com/riywo/loginshell v0.0.0-20200815045211-7d26008be1ab
      dep github.com/sergi/go-diff  v1.2.0
      dep github.com/skeema/knownhosts  v1.1.1
      dep github.com/spf13/cobra  v1.7.0
      dep github.com/spf13/pflag  v1.0.5
      dep github.com/xanzy/ssh-agent  v0.3.3
      dep github.com/xo/terminfo  v0.0.0-20220910002029-abceb7e1c41e
      dep golang.org/x/crypto v0.9.0
      dep golang.org/x/net  v0.10.0
      dep golang.org/x/sys  v0.8.0
      dep golang.org/x/term v0.8.0
      dep golang.org/x/text v0.9.0
      dep gopkg.in/warnings.v0  v0.1.2
      dep gopkg.in/yaml.v3  v3.0.1
      build -compiler=gc
      build -tags=netgo
      build -trimpath=true
      build CGO_ENABLED=0
      build GOARCH=arm64
      build GOOS=darwin
 [d]  installed at: /Users/joey/.nix-profile/bin
 [d]  git autopush: false
 [d]  git autocommit: false
 [d]  git autopull: false
 [*]  Apply fleek configuration
  ERROR
  ERROR
          Error: configuration files not found, run `fleek init`
  ERROR
 [d]
      ExecutionID:538de73f2a3e469f8a8884af3dfe4c86

      github.com/ublue-os/fleek/internal/fleekcli/usererr.New
        github.com/ublue-os/fleek/internal/fleekcli/usererr/usererror.go:26
      github.com/ublue-os/fleek/internal/fleekcli.mustConfig
        github.com/ublue-os/fleek/internal/fleekcli/root.go:180
      github.com/ublue-os/fleek/internal/fleekcli.apply
        github.com/ublue-os/fleek/internal/fleekcli/apply.go:35
      github.com/ublue-os/fleek/internal/fleekcli.ApplyCommand.func1
        github.com/ublue-os/fleek/internal/fleekcli/apply.go:24
      github.com/spf13/cobra.(*Command).execute
        github.com/spf13/cobra@v1.7.0/command.go:940
      github.com/spf13/cobra.(*Command).ExecuteC
        github.com/spf13/cobra@v1.7.0/command.go:1068
      github.com/spf13/cobra.(*Command).Execute
        github.com/spf13/cobra@v1.7.0/command.go:992
      github.com/ublue-os/fleek/internal/midcobra.(*midcobraExecutable).Execute
        github.com/ublue-os/fleek/internal/midcobra/midcobra.go:67
      github.com/ublue-os/fleek/internal/fleekcli.Execute
        github.com/ublue-os/fleek/internal/fleekcli/main.go:33
      github.com/ublue-os/fleek/internal/fleekcli.Main
        github.com/ublue-os/fleek/internal/fleekcli/main.go:24
      main.main
        github.com/ublue-os/fleek/cmd/fleek/main.go:8
      runtime.main
        runtime/proc.go:250
      runtime.goexit
        runtime/asm_arm64.s:1172
```

**Comment by tml** [comment]
![image](https://github.com/ublue-os/fleek/assets/35148/df540d67-92ca-4ed4-8eb0-5dd40aba5308)

**Comment by bketelsen** [comment]
my hypothesis is that the 'ssh' config in the yaml that you added has bad syntax, which causes everything after that 'ssh' key to be ignored because it isn't valid yaml.  The .fleek.yaml file has a key that stores the relative location of the configuration file, which is where fleek will look to load your config. Since that key wasn't found fleek doesn't know where to look. That is a bug that should be fixed by a default value.

**Comment by bketelsen** [comment]
Default flake location fixed in #214
